src/main/webapp/app/exercises/text/assess-new/conflicts/text-feedback-conflicts.component.html

```diff
@@ -0,0 +1,81 @@
+<jhi-text-feedback-conflicts-header
+    [numberOfConflicts]="conflictingSubmissions?.length"
+    (didChangeConflictIndex)="didChangeConflictIndex($event)"
+    [haveRights]="canOverride(this.leftSubmission?.result)"
+    [overrideBusy]="overrideBusy"
+    [markBusy]="markBusy"
+    [isOverrideDisabled]="isOverrideDisabled"
+    [isMarkingDisabled]="isMarkingDisabled"
+    (overrideLeftSubmission)="overrideLeftSubmission()"
+    (discardConflict)="discardConflict()"
+></jhi-text-feedback-conflicts-header>
+
+<ng-container *ngIf="conflictingSubmissions && conflictingSubmissions.length > 0; then conflict; else noConflict"></ng-container>
+
+<ng-template #conflict>
+    <div class="conflict-container">
+        <div class="left card" [class.border-0]="false">
+            <div class="card-header bg-primary text-white">
+                <div class="flew-grow-1">
+                    <span *ngIf="isAssessor(this.leftSubmission?.result); else otherAssessorLeftCard" jhiTranslate="artemisApp.textAssessment.conflict.assessor"></span>
+                    <ng-template #otherAssessorLeftCard>
+                        <span
+                            jhiTranslate="artemisApp.textAssessment.conflict.otherAssessor"
+                            [translateValues]="{ otherUser: this.leftSubmission?.result?.assessor?.name + ' (' + this.leftSubmission?.result?.assessor?.login + ')' }"
+                        ></span>
+                    </ng-template>
+                </div>
+                <jhi-score-display [maxScore]="exercise?.maxScore" [score]="leftTotalScore"></jhi-score-display>
+            </div>
+            <div class="left-body card-body" [class.p-0]="false">
```

Comment:
Why is `[class.p-0]="false"` needed?

Reply:
so it is `false` by default. All are removed 👍